mautrix/bridge/commands/handler.py

```diff
@@ -136,11 +137,16 @@ def reply(self, message: str, allow_html: bool = False, render_markdown: bool =
         html = self._render_message(message, allow_html=allow_html,
                                     render_markdown=render_markdown)
 
-        return self.az.intent.send_notice(self.room_id, message, html=html)
+        if self.is_portal:
+            portal = await self.processor.bridge.get_portal(self.room_id)
+            return await portal.main_intent.send_notice(self.room_id, message, html=html)
+        else:
+            return await self.az.intent.send_notice(self.room_id, message, html=html)
 
     def mark_read(self) -> Awaitable[None]:
         """Marks the command as read by the bot."""
-        return self.az.intent.mark_read(self.room_id, self.event_id)
+        if not self.is_portal:
+            return self.az.intent.mark_read(self.room_id, self.event_id)
```

Comment:
Don't know if we should do this, as this probably makes the bot not read stuff in encrypted direct chats?
I just wanted to prevent the complaining about the bot not being in my rooms.

```suggestion
        return self.az.intent.mark_read(self.room_id, self.event_id)
```